### PR TITLE
Android13 notification

### PR DESCRIPTION
**Pull request by 2giwon**

### CODE REVIEW

File: Android13/Android13_Notification.md [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+- 모든 알림 채널이 차단됨
+- 시스템 설정에서 알림을 수동으로 끄는 것과 같은 동작
+
+<aside>
+💡 **Caution : 앱이 12L 이하이고 허용안함을 누르면 아래와 같은 상황이 발생되기 전까지 Dialog가 다시 표시 되지 않음. 
+
+- 사용자가 재설치하기 전
+- Android 13 대상으로 앱을 업데이트**
+
+</aside>
```
Comment (BeokBeok) [inline Android13/Android13_Notification.md]: 여기 스타일이 깨져요 😭

File: Android13/Android13_Notification.md [hunk 2]
```diff
@@ -0,0 +1,179 @@
...
+- Android 13으로 업데이트 시 모든 적격 앱에 권한을 자동으로 부여함
+- 이런 앱은 계속해서 사용자에게 알림을 보낼 수 있으며
+- 런타임 Permission Dialog는 표시 되지 않음
+
+<aside>
+💡 사용자가 12L 에서 앱을 설치했고 알림을 허용한 경우 
+
+시스템은 13으로 업데이트 했을 때 앱의 알림 권한을 자동으로 부여함
+
+</aside>
```
Comment (BeokBeok) [inline Android13/Android13_Notification.md]: 여기도 스타일이 깨집니다 😭

**Review by BeokBeok — APPROVED** [review]
좋은 글 잘 봤습니다!

**Review by GongSulMin — APPROVED** [review]

File: Android13/Android13_Notification.md (hunk 1 above)
Comment (2giwon) [reply Android13/Android13_Notification.md]: 수정했습니다 [bfc53e4](https://github.com/JoyOfAndroid/study/pull/43/commits/bfc53e4741660fbcf7f5b73cc18c9147839a9d2b)

File: Android13/Android13_Notification.md (hunk 2 above)
Comment (2giwon) [reply Android13/Android13_Notification.md]: 수정했습니다 [bfc53e4](https://github.com/JoyOfAndroid/study/pull/43/commits/bfc53e4741660fbcf7f5b73cc18c9147839a9d2b)

**Pull request merged**